WooCommerce/Classes/ViewRelated/Orders/Order Details/Order Summary Section/SummaryTableViewCell.swift

```diff
@@ -7,6 +7,61 @@ struct SummaryTableViewCellPresentation {
     let statusName: String
 }
 
+/// The ViewModel for `SummaryTableViewCell`.
+///
+/// TODO This and that cell class should be renamed to be less ambiguous.
+///
+struct SummaryTableViewCellViewModel {
+    private let billingAddress: Address?
+    private let dateCreated: Date
+    private let orderNumber: String
+
+    let presentation: SummaryTableViewCellPresentation
+
+    private let calendar: Calendar
+
+    init(order: Order,
+         status: OrderStatus?,
+         calendar: Calendar = .current) {
+
+        billingAddress = order.billingAddress
+        dateCreated = order.dateCreated
+        orderNumber = order.number
+
+        presentation = SummaryTableViewCellPresentation(
+            status: status?.status ?? OrderStatusEnum(rawValue: order.statusKey),
+            statusName: status?.name ?? order.statusKey
+        )
+
+        self.calendar = calendar
+    }
+
+    /// The full name from the billing address
+    ///
+    var billedPersonName: String {
+        if let billingAddress = billingAddress {
+            return "\(billingAddress.firstName) \(billingAddress.lastName)"
```

Comment:
shall we localize this string? in certain languages (like Mandarin), the name is in the inverse order and without space in between (`"\(billingAddress.lastName)\(billingAddress.firstName)"`) 🙂

Reply:
Thanks for letting me know! Localized in bb2688f9e471210f3d5d610b6ed5d3319d0ad4b4.

Reply:
(same as the other comment)